.github/workflows/bcc-test.yml

```diff
@@ -94,6 +94,86 @@ jobs:
         name: critical-tests-${{ matrix.env['TYPE'] }}-${{ matrix.os.version }}
         path: tests/python/critical.log
 
+  test_bcc_fedora:
+    runs-on: ubuntu-20.04
```

Comment:
test_bcc_fedora and runs-on ubuntu-20.04? I do see we are using `Dockerfile.fedora` below. So `ubuntu-20.04` is a copy-paste error?

Reply:
No, this is intentional. `ubuntu-20.04` here describes the OS of the "GitHub actions runner" VM. List of supported runner OS's [here](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) doesn't include fedora.

The docker container is Fedora, though. So aside from the kernel still being `ubuntu-20.04`'s, tests are running using userspace and toolchains of Fedora.

Reply:
for ubuntu run:
>  Could NOT find LuaJIT (missing: LUAJIT_LIBRARIES LUAJIT_INCLUDE_DIR) 
luajit package may be missing in the system. Will need to figure how to get the package properly.

Reply:
> No, this is intentional. `ubuntu-20.04` here describes the OS of the "GitHub actions runner" VM. List of supported runner OS's [here](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) doesn't include fedora.
> 
> The docker container is Fedora, though. So aside from the kernel still being `ubuntu-20.04`'s, tests are running using userspace and toolchains of Fedora.

Okay, utunbu vm and fedora docker container. Make sense. thanks for explanation.